cpp/foxglove/include/foxglove/server.hpp

```diff
@@ -62,9 +65,12 @@ struct WebSocketServerCallbacks {
 };
 
 struct WebSocketServerOptions {
+  friend class WebSocketServer;
+
+  Context context;
   std::string name;
-  std::string host;
-  uint16_t port;
+  std::string host = "127.0.0.1";
+  uint16_t port = 0;
```

Comment:
Python and Rust default to port 8765, which matches the default connection setting in the app; can we do the same here?